engine/classes/Elgg/PasswordService.php

```diff
@@ -89,7 +89,8 @@ function sendNewPasswordRequest($user_guid) {
 
 		// generate link
 		$link = _elgg_services()->config->getSiteUrl() . "changepassword?u=$user_guid&c=$code";
```

Comment:
I didn't want to hack my way through the password service. The code is injected in the password reset form and then validated. There would be BC implications removing it entirely.